tests/fixtures/no-color/multiple_annotations.svg

```diff
@@ -20,15 +20,15 @@
 </tspan>
     <tspan x="10px" y="46px"><tspan>    |</tspan>
 </tspan>
-    <tspan x="10px" y="64px"><tspan> 96 | fn add_title_line(result: &amp;mut Vec&lt;String&gt;, main_annotation: Option&lt;&amp;Annotation&gt;) {</tspan>
+    <tspan x="10px" y="64px"><tspan>96  | fn add_title_line(result: &amp;mut Vec&lt;String&gt;, main_annotation: Option&lt;&amp;Annotation&gt;) {</tspan>
```

Comment:
rustc left-aligns numbers?  That is an odd choice and want to make sure we make this choice knowingly (and considering whether we should deviate)

Reply:
This is also a change that is separate from the commit title / PR

Reply:
They were left-aligned in the tests I ported. Either way, I removed that from this PR and will create a PR just for it